src/deluge/gui/l10n/seven_segment.cpp

```diff
@@ -338,6 +338,11 @@ PLACE_SDRAM_DATA Language seven_segment{
         {STRING_FOR_DEFAULT_UI_KEYBOARD_LAYOUT_INKEY, "INKY"},
         {STRING_FOR_DEFAULT_UI_SONG, "SONG"},
         {STRING_FOR_DEFAULT_UI_SONG_LAYOUT_ROWS, "ROWS"},
+        {STRING_FOR_DEFAULT_UI_DEFAULT_STARTUP_SONG_MODE, "STSONG"},
```

Comment:
since screen is only 4 chars, would you say SONG instead? as we are already in the Defaults submenu, there is enough context, to suppose SONG is going to be a setting for Default Song

Reply:
updated thx monitus, i ll play around with the 7seg emulator to have more clue on 7seg limitations